packages/web-components/src/components/ArrowButton/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { clsx } from "clsx";
+import type { FunctionalComponent } from "preact";
+import { SvgIcon, allureIcons } from "@/components/SvgIcon";
+import styles from "./styles.scss";
+
+export interface ArrowButtonProps {
+  isOpened?: boolean;
+  iconSize?: "m" | "xs" | "s";
+  buttonSize?: "m" | "xs" | "s";
+  className?: string;
+  icon?: string;
+}
+
+export const ArrowButton: FunctionalComponent<ArrowButtonProps> = ({
+  isOpened,
+  buttonSize = "m",
+  iconSize = "xs",
+  className,
+  icon = allureIcons.lineArrowsChevronDown,
+  ...rest
+}) => {
+  return (
+    <button className={clsx(styles["arrow-button"], styles[`arrow-button-${buttonSize}`])} {...rest}>
```

Comment:
Move `rest` before `className`. Add `type="button"` to the element to prevent potential form submitting